configs/config.json.cluster.example

```diff
@@ -22,23 +22,6 @@
         "type": "HandleSoftBan"
       },
       {
-        "type": "CompleteTutorial",
```

Comment:
Make sure how? It generates a new one if the first is not accepted, should it keep generating and trying till one is accepted?